openapi-docs/README.md

```diff
@@ -0,0 +1,145 @@
+# ambianic_functions_collection
```

Comment:
@vickywane I did not see a reference to this document in the ghpages published version. I only see the REST API, but not the JS examples.

Reply:
I can't find the configuration to link this document to the files generated by `bootprint`. [Bootprint](https://www.npmjs.com/package/bootprint-openapi) generates the static docs from the `openapi-spec.yaml` file.

Reply:
@vickywane How does bootprint compare to the [openapi-generator tool docs](https://openapi-generator.tech/docs/generators/#documentation-generators) output?

If there is no way to generate one file with all API docs linked, then we can create a section in the main docs under Developer Guide / Premium Services and there link to the various API and SDK client docs. See how we do that for the Ambianic Edge and Peer-to-Peer Communication sections.

In `mkdocs.yml`
```
...
      - Ambianic UI: https://github.com/ambianic/ambianic-ui
      - Ambianic Edge: 
        - Python API Reference: https://ambianic.github.io/ambianic-edge/ambianic/
        - Gtihub repo: https://github.com/ambianic/ambianic-edge
      - Ambianic Box: https://github.com/ambianic/ambianic-box
      - Fall Detection: https://github.com/ambianic/fall-detection
      - Peer to Peer Communication: 
        - p2p architecture: https://webrtchacks.com/private-home-surveillance-with-the-webrtc-datachannel/
        - peerjs-python: https://github.com/ambianic/peerjs-python
        - ambianic-pnp: https://github.com/ambianic/ambianic-pnp
```


![Screen Shot 2021-06-01 at 6 23 52 PM](https://user-images.githubusercontent.com/2234901/120401918-99a6d600-c306-11eb-96eb-4c0e2f04264d.png)



![Screen Shot 2021-06-01 at 6 23 46 PM](https://user-images.githubusercontent.com/2234901/120401921-9ad80300-c306-11eb-9d2a-957a5f36cb21.png)